spec/requests/internal/broadcasts_spec.rb

```diff
@@ -48,6 +48,20 @@
       end
     end
 
+    describe "PUT /internal/broadcasts" do
+      let!(:broadcast) { create(:welcome_broadcast, active: false) }
+
+      it "updates the Broadcasts last_active_at timestamp" do
+        old_time = broadcast.last_active_at
+        Timecop.freeze(Time.current) do
+          expect do
+            put "/internal/broadcasts/#{broadcast.id}", params: params
+          end.to change { broadcast.reload.active }.from(false).to(true)
+          expect(broadcast.last_active_at).not_to eq(old_time)
```

Comment:
Can we assert the the `last_active_at` is the same as `Time.current`? Rather than asserting that it's just _not_ the same as what it was?

Reply:
Great question @vaidehijoshi!

I attempted to do this multiple times and ways yesterday, and while the specs always passed locally, they always failed on their Travis builds due to the _slightest_ (I'm talking milliseconds) discrepancy in timestamps. At this point in time, I don't think it's worth the time it takes for me to run this through Travis again because it seems like the odds of it passing are relatively low and I don't want to hog the build pipeline again. 😬 😅